.env

```diff
@@ -14,7 +14,8 @@ GRAFANA_VERSION=6.3.5
 PROMETHEUS_VERSION=2.12.0
 
 # NeoFS images versions
-NEOFS_IMAGES_VERSION=0.1.11
+NEOFS_IMAGES_VERSION=0.2.3
+NEOFS_CLI_VERSION=0.2.1
```

Comment:
not that obvious from the commit why CLI and IMAGES now have different versions.

Reply:
In the development we are trying to separate neofs-node repo into group of small libraries and applications. 0.2.x releases support `neofs-proto` library, that allows to build applications for NeoFS, e.g. NeoFS CLI. 

Do you think we need explicit mentions about this?

Reply:
at least in commit message it will be nice to have, thank you! =)

Reply:
Alright! I will add more info about this in commit message and force-push the changes into this branch.